### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by pkothare**
- Shortened sentences to be more impactful
- Fixed grammar and language across the board
- Added missing links for references to tools, libraries, packages etc.
- Updated contributing section to provide more clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Guide to Easy Dynamics GitHub repositories
 
 The guide to Easy Dynamics GitHub repositories is a [GitHub Pages](https://pages.github.com/) project that helps visitors and contributors 
-navigate the software projects at Easy Dynamics. This readme is primarily to orient contributors for this repository. If you are a visitor 
-or contributor at Easy Dynamics please visit the published version of this guide at [easydynamics.github.io](https://easydynamics.github.io/) 
-to get further details about procedures and any other  to get started.
+navigate software projects at Easy Dynamics. The purpose of this readme is to orient contributors of **this repository**. If you're new to software development Easy Dynamics, please visit the published version of this guide at [easydynamics.github.io](https://easydynamics.github.io/)
```
Comment (laurelmay) [inline README.md]:
```suggestion
navigate software projects at Easy Dynamics. The purpose of this readme is to orient contributors of **this repository**. If you're new to software development at Easy Dynamics, please visit the published version of this guide at [easydynamics.github.io](https://easydynamics.github.io/) 
```

File: README.md [hunk 2]
```diff
@@ -12,15 +11,14 @@ to get further details about procedures and any other  to get started.
...
\ No newline at end of file
+If you notice a bug or want to report an issue with [easydynamics.github.io](https://easydynamics.github.io/), see if it's related to any of the [existing ones](https://github.com/EasyDynamics/easydynamics.github.io/issues) and join the conversation. If not, open a new issue and describe it for us. If you want to fix minor bugs or issues, feel free to open a pull request with the appropriate changes. For broader topics, start a conversation on an issue and build consensus with the maintainers before creating a pull request. Last but not least please follow [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
```
Comment (laurelmay) [inline README.md]:
```suggestion
If you notice a bug or want to report an issue with [easydynamics.github.io](https://easydynamics.github.io/), see if it's related to any of the [existing ones](https://github.com/EasyDynamics/easydynamics.github.io/issues) and join the conversation. If not, open a new issue and describe it for us. If you want to fix minor bugs or issues, feel free to open a pull request with the appropriate changes. For broader topics, start a conversation on an issue and build consensus with the maintainers before creating a pull request. Last but not least, please follow [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
```

**Review by laurelmay — CHANGES_REQUESTED** [review]

**Review by rgauss — COMMENTED** [review]
Look good, pending @kylelaker's suggested changes.

File: README.md (hunk 2 above)
Comment (pkothare) [reply README.md]: Accidentally hit commit suggestion instead of adding it to the batch. Will fix this comma on a another commit.

**Pull request merged**